templates/repo/issue/view_content/comments.tmpl

```diff
@@ -111,11 +110,9 @@
 		{{else if eq .Type 2}}
 			<div class="timeline-item event" id="{{.HashTag}}">
 				<span class="badge bg-red text-white">{{svg "octicon-circle-slash"}}</span>
-				<a href="{{.Poster.HomeLink}}">
-					{{avatar .Poster}}
-				</a>
-				<span class="text grey">
-					<a class="author" href="{{.Poster.HomeLink}}">{{.Poster.GetDisplayName}}</a>
+				{{template "shared/user/avatarlink" .Poster}}				<span class="text grey">
+					{{template "shared/user/authorlink" .Poster}}
+
```

Comment:
Why the whitespace between the first template and the `span`?
Why the empty line?
And why do I see that combination so often that we can extract it into its own template?

Reply:
> Why the whitespace between the first template and the `span`?

It's a search/replace problem - now I know it's there I can do another search/replace to fix it.

> Why the empty line? 

There isn't one it's how the github is rendering the change.

> And why do I see that combination so often that we can extract it into its own template?

The reuse is not a complete element and to really refactor to make it a complete html element  it we need `$.locale.Tr` to become `Tr` (i.e. a function rather than an object.)

Reply:
Eurgh it looks that isn't going to work... The problem appears to be that the funcMap is shared across the templates and executions and there isn't really a way to make functions refer to some current render context as far as I can see. 

Although we have the function `dict` it's somewhat unsatisfying and quite inefficient to use.

Maybe we need something that just wraps the `$` but I'm not sure.